### ISSUE TITLE
文本检测模型无法收敛

### ISSUE THREAD

**Issue by 1191658517** [PaddlePaddle/PaddleOCR#8496]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu18.04
- 版本号/Version：Paddle：2.3.2   PaddleOCR2.6.1.0： 

您好，我用trdg合成了一批如以下所示的图片（超过10M的没有放进来），图片宽高像素从几百到几千不等，然后利用PPOCRV2的det的server模型预训练模型微调后效果非常差。训练了150epoch，hmean只有0.36，预测结果也不好，能请教一下可能是什么原因吗？感谢

![background_img11](https://user-images.githubusercontent.com/54796134/204759214-af7ec797-1505-4aab-9e3d-ca048a104f63.png)
![background_img87](https://user-images.githubusercontent.com/54796134/204760945-a7652e24-3346-4025-8773-73b3be84d483.png)
![background_img102](https://user-images.githubusercontent.com/54796134/204760148-98fbc42c-4a77-4ae1-8474-d9b02632e539.png)

以下是我的配置
Global:
  use_gpu: True
  epoch_num: 1000
  log_smooth_window: 20
  print_batch_step: 22
  save_model_dir: ./output/ch_db_res18-1130-SysOnly/
  save_epoch_step: 164

  eval_batch_step:
    - 0
    - 3000
  cal_metric_during_train: False
  pretrained_model: ./pretrain_models/ch_ppocr_server_v2.0_det_train/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: True
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/1130-SysOnly/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    layers: 18
    disable_se: True
    freeze_params: True
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.3
  max_candidates: 1000
  unclip_ratio: 2.0
  use_polyon: True
  score_mode: slow

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/20221129_det_ppocr_sysOnly
    label_file_list:
      - ./train_data/20221129_det_ppocr_sysOnly/train.txt
    ratio_list: 1.0
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - EastRandomCropData:
          size: [1024, 1024]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 2
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 4
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/20221129_det_ppocr_sysOnly
    label_file_list:
      - ./train_data/20221129_det_ppocr_sysOnly/test.txt
    ratio_list: 1.0
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
           limit_side_len: 1920
           limit_type: 'max'
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2
    user_shared_memory: True
